hphp/hack/src/typing/typing_visibility.ml

```diff
@@ -159,16 +159,19 @@ let check_typedef_access ~use_pos ~in_signature env td =
     internal
     td.td_module
 
-let is_visible_for_obj env vis =
+let is_visible_for_obj ?(is_method = false) env vis =
```

Comment:
This should probably be a compulsory argument, which will help you spot any other call sites that need to be updated.

```suggestion
let is_visible_for_obj ?(is_method) env vis =
```